requirements.txt

```
@@ -4,7 +4,7 @@
 
 ciso8601~=2.3.0 # To speed up influxDB queries
 geopy~=2.3.0
-openstef~=3.4.1
+openstef~=3.4.2
```

```suggestion
openstef~=3.4.3
```

This should be version 3.4.3 as that is the OpenSTEF version you just realised.